site/_data/lambdas.yaml

```diff
@@ -120,3 +120,8 @@
   docLink: /docs/v1/tech/lambdas/scim-user-response-converter
   version: 1.36.0
   menuIcon: fa-server fas
+- displayName: Self-Service Registration
+  typeText: SelfServiceRegistrationValidation
+  docLink: /docs/v1/tech/lambdas/self-service-registration
+  version: 1.43.0
+  menuIcon: fa-ballot-check fas
```

Comment:
I think this is the only one that uses `fa-ballot-check`. Do we want to add a new icon? Or is there another?

Reply:
This may be fine. I just want to make sure we are choosing the right icon if we are adding a new one. Are there other lambdas that we may add later that would make sense with this same icon?